docs/building-from-source.md

```diff
@@ -61,9 +61,9 @@ If you check out repos manually then it is your responsibility to ensure that th
 ```
 mkdir repos
 git -C repos clone https://github.com/llvm/llvm-project.git
-git -C repos/llvm-project am -k ../../patches/llvm-project/*.patch
+git -C repos/llvm-project am -k `pwd`/../patches/llvm-project/*.patch
```

Comment:
The backticked `pwd` invocation ought to be in double quotes, to protect against special characters like spaces in the output of the subprocess.

Also it may be possible to avoid the subprocess invocation completely, by using `"$PWD"`.

(If you do have to use a subprocess, my preference is to use `$(...)` rather than backticks, but the main reason is that they nest more sensibly, which isn't particularly important here. Even so, if it were me, I'd try to model good habits at every opportunity.)

Reply:
Great points! Fixed.